src/stores/results.ts

```diff
@@ -1,13 +1,49 @@
 import { defineStore } from "pinia"
 import { useStore } from "@/stores/index.js"
-import { StandardBenefit } from "@data/types/benefits"
+import { StandardBenefit, BenefitGroup } from "@data/types/benefits"
+import { hasBafaInterestFlag } from "@/lib/benefits.js"
+import ABTestingService from "@/plugins/ab-testing-service.js"
 
 export const useResultsStore = defineStore("results", {
   getters: {
     benefits(): StandardBenefit[] {
-      return this.resultats?.droitsEligibles
+      return this.resultats?.droitsEligibles || []
     },
-    hasBenefits() {
+    benefitTreeGroupExperiment(): (StandardBenefit | BenefitGroup)[] {
+      const groups = this.benefits.reduce(
+        (groups, benefit) => {
+          if (hasBafaInterestFlag(benefit)) {
+            groups.bafa.push(benefit)
+          } else {
+            groups.other.push(benefit)
+          }
+          return groups
+        },
+        { bafa: [], other: [] }
+      )
+      if (groups.bafa.length) {
+        const bafaGroup: BenefitGroup = {
```

Comment:
il faudrait gérer le cas où il n'y a qu'une seule aide BAFA et dans ce cas simplement conserver la liste des aides `this.benefits`.